H_min: 37
H_max: 79
S_min: 40
S_max: 219
V_min: 5
V_max: 91
A_min: 0
A_max: 50000
gaussian_ksize: 0
gaussian_sigma: 0
morph_size: 0
invert_Hue: false
